acspy/acsc.py

```diff
@@ -58,48 +67,116 @@
 COUNTERCLOCKWISE = 1
 CLOCKWISE = -1
 INT_BINARY = 4
-REAL_BINARY	 = 8
+REAL_BINARY = 8
 INT_TYPE = 1
 REAL_TYPE = 2
+DEFAULT = -1
+INFINITE = -1
+
+ax_mflags = {
+        'DUMMY': 0,
```

Comment:
```suggestion
    "DUMMY": 0,
```

For consistency, these should be switched to double quotes, and only indented 4 spaces.